### ISSUE TITLE
new "multiplayer" dms don't seem to get handled correctly

### ISSUE THREAD

**Issue by sandsmark** [ltworf/localslackirc#77]
when new #mpdm-foo channels are created, they don't seem to open here, so messages in them seem to just come directly in PMs from the users writing in them.

**Comment by sandsmark** [comment]
```
{  
   'type':'group_joined',
   'channel':{  
      'id':'F00BAR',
      'name':'mpdm-foo--baz--baz-1',
      'is_channel':False,
      'is_group':True,
      'is_im':False,
      'created':1535526230,
      'is_archived':False,
      'is_general':False,
      'unlinked':0,
      'name_normalized':'mpdm-foo--bar--baz-1',
      'is_shared':False,
      'creator':'B4RB4Z',
      'is_ext_shared':False,
      'is_org_shared':False,
      'shared_team_ids':[  
         'B4ZB4R'
      ],
      'pending_shared':[  

      ],
      'is_pending_ext_shared':False,
      'is_member':True,
      'is_private':True,
      'is_mpim':True,
      'last_read':'0000000000.000000',
      'latest':None,
      'unread_count':0,
      'unread_count_display':0,
      'is_open':False,
      'members':[  
         'F00000000',
         'BAAAAAAAR',
         'BAZZZZZZZ'
      ],
      'topic':{  
         'value':'Group messaging',
         'creator':'F0000000',
         'last_set':1535526230
      },
      'purpose':{  
         'value':'Group messaging with: @foo @bar @baz',
         'creator':'F0000000',
         'last_set':1535526230
      },
      'priority':0
   }
}
```

**Comment by ltworf** [comment]
Should be fixed in c1c7a9d298e819dd67b2f228848e50e08a100ba0
